test/SerialIOTest.cpp

```diff
@@ -1,11 +1,11 @@
 #define BOOST_TEST_MODULE libopenpmd_serial_io_test
 
-
-#include <boost/test/included/unit_test.hpp>
-
 /* make Writable::parent visible for hierarchy check */
 #define protected public
-#include "Series.hpp"
+#include "openPMD/openPMD.hpp"
+#undef protected
```

Comment:
This little detail is very much appreciated.

Reply:
hurray :)